.github/CHANGELOG.md

```diff
@@ -424,10 +457,10 @@ and requirements-ci.txt (unpinned). This latter would be used by the CI.
 
 * The QFT operation is moved to template
   [(#1548)](https://github.com/PennyLaneAI/pennylane/pull/1548)
-  
-* The `qml.ResetError` is now supported for `default.mixed` device. 
+
+* The `qml.ResetError` is now supported for `default.mixed` device.
   [(#1541)](https://github.com/PennyLaneAI/pennylane/pull/1541)
-  
+
 * `QNode.diff_method` will now reflect which method was selected from `diff_method="best"`.
   [(#1568)](https://github.com/PennyLaneAI/pennylane/pull/1568)
 
```

Comment:
I can't comment, but there are a couple of cases below where 'user' is left in the changelog entry. I would re-write them to remove usage of 'user'.

E.g.,

> An error is now raised when no arguments are passed to an observable, to inform that wires have not been supplied. (#1547)

> QNodes now include validation to warn ~users~ if a supplied keyword argument is not one of the recognized arguments. (#1496)

Reply:
Done.